ee/src/main/java/org/jboss/as/ee/component/ProxyInvocationHandler.java

```diff
@@ -48,7 +48,7 @@
     /**
      * Construct a new instance.
      *
-     * @param interceptors the interceptors map to use
+     * @param Jakarta Interceptors the Jakarta Interceptors map to use
```

Comment:
Please be aware of the javadoc syntax; the `interceptors` just after `@param` can't be changes as it must match the actual parameter name in the method that follows this javadoc block.

Reply:
That was a mistake! I'll change it :)